examples/aggregation.py

```diff
@@ -38,12 +36,9 @@ def __str__(self):
 
 
 async def run():
-    client = SqliteClient('example_aggregation.sqlite3')
-    await client.create_connection()
-    Tortoise.init(client)
-    await generate_schema(client)
-
-    tournament = Tournament(name='New Tournament')
+    await Tortoise.init(config_file='config.json')
```

Comment:
Could we please have this split up?
I mean `Tortoise.init()` should instead recieve configuration in a dict, and for cases where we want to have support of config files or enviromment variables should be done as a layer above that.

This will make it easier to wrap the ORM in another framework.

Reply:
Well - it already receives config as dict as main method of init - look for postgres example. 
`config_file` is just alternative option for those who use tortoise directly

Reply:
:+1: